CHANGELOG.rst

```diff
@@ -9,20 +9,30 @@ Features
 --------
 
 * Add message table rotation for webpush users. Issue #191.
+* New Bridge HTTP API. Issues #238, #250, #251
+  In cooperation with the GCM client work the HTTP Bridge API has been
+  simplified. The new method has been detailed in /api/endpoint.py.
+  In essence: The API is now bearer token based, and uses the form
+  /v1/{BridgeType}/{BridgeToken}/registration[/{uaid}/[subscription/[{chid}]]]
 
 Bug Fixes
 ---------
+* Fixed some issues around the Bridge API delete functions.
 
 * Relax endpoint TLS cert requirement for https scheme. Issue #249.
 
 WebPush
 -------
 
+Backwards Incompatibilities
+---------------------------
+* The previous Bridge HTTP API has been immediately deprecated.
```

Comment:
Has this been removed? If it's deprecated, its not backward incompatible... yet.